### PR TITLE
GitCommitBear: Change python -> python3

### PR DESCRIPTION
**Pull request by Asnelchristian**
The command python -m nltk.downloader punkt
was modify to take python3

Fixes https://github.com/coala-analyzer/coala-bears/issues/454

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -133,7 +133,7 @@ work properly.)
 Project Status
 --------------
 
-|Linux Build Status| |Windows Build status| |OSX Build status|
+|Linux Build Status| |Windows Build status|
```
Comment (Makman2) [inline README.rst]: wasn't this already removed? Is a rebase missing?

Comment (Makman2) [inline README.rst]: yeah indeed just seeing^^

Comment (Adrianzatreanu) [inline README.rst]: thats my commit, this is weird, thats why my autorship shows there

Comment (hypothesist) [inline README.rst]: He amended to the last commit in the git log?

Comment (Adrianzatreanu) [inline README.rst]: it might be.

File: bears/vcs/git/GitCommitBear.py [hunk 2]
```diff
@@ -157,7 +157,7 @@ def check_imperative(self, paragraph):
                 return None
         except LookupError as error:  # pragma: no cover
             self.err("NLTK data missing, install by running following commands "
-                     "`python -m nltk.downloader punkt"
+                     "`python3 -m nltk.downloader punkt"
                      " maxent_treebank_pos_tagger averaged_perceptron_tagger`")
             return
```
Comment (Asnelchristian) [inline bears/vcs/git/GitCommitBear.py]: i am sorry, i used git commit --amend that is why the other one is there

**Comment by Adrianzatreanu** [comment]
You ammended and took an old commit with this, please fix it :)

**Comment by Adrianzatreanu** [comment]
unack 364bb2d

**Comment by Asnelchristian** [comment]
what should i modify again ?

**Comment by Adrianzatreanu** [comment]
nothing yet, let me see if the tests pass now.

**Comment by Adrianzatreanu** [comment]
I do not know for sure, you might need to do a ``git pull origin master`` but i'm not 100% sure this is the fix, sorry ^^

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/461/commits/0cc29c48a5608f7e132fc0b5ee436a8eabc03a6c

**Comment by sils1297** [comment]
Hi,

your commit looks good. However, it looks like your PR is out of date and needs a rebase. [This page](http://coala.readthedocs.org/en/latest/Users/Tutorials/Git_Help.html) may help you to get started on this. (*Do not press the update branch button. It's evil.*)

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala-analyzer/coala.git
```

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

Ping us on [our gitter channel](https://gitter.im/coala-analyzer/coala) if you need help!

**Comment by sils1297** [comment]
(that should also fix your tests)

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/461/commits/21877a1da5d1c1505834c1994390c75c78dcd33a

**Comment by sils1297** [comment]
ok codecov is crazy again, we'll be ignoring it :) this contribution is cool! @Asnelchristian it's your first contribution? Welcome to the team! It's awesome that you're with us! You're now officially a junior-coalaian. I'm not sure if I pointed this out but it might be interesting to you: https://github.com/coala-analyzer/coala/wiki/coala-Steam-Offer

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
@Asnelchristian rultor will make a fastforward (no merge commit, clean linear history) and will automatically make a development release so if you install coala via `pip` with the `--pre` flag you'll see that your changes are already released after this is merged.

**Pull request merged**